docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -1,29 +1,37 @@
 ---
 id: eks-terraform
-title: "Deploy an EKS cluster with Terraform"
+title: "Deploy an EKS cluster with Terraform (stable)"
```

Comment:
I'd not add the `(stable)`, same as eksctl either tags.

What is stable even referring to and why isn't the rest stable?